src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -82,54 +85,72 @@
   </div>
   <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
     <dt class="govuk-summary-list__key">
-      Load sensing valve
+      Suspension type
     </dt>
-    <dd id="test-loadSensingValve" class="govuk-summary-list__value">
-      <div *ngIf="!!activeRecord.brakes; else hyphen">
-        {{ activeRecord.brakes.loadSensingValve | DefaultNullOrEmpty }}
-      </div>
+    <dd id="test-suspensionType" class="govuk-summary-list__value">
+      {{ activeRecord.suspensionType | DefaultNullOrEmpty }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Antilock braking system
+      Road friendly suspension
     </dt>
-    <dd id="test-antilockBrakingSystem" class="govuk-summary-list__value">
-      <div *ngIf="!(activeRecord.brakes == null); else hyphen">
-        {{ activeRecord.brakes.antilockBrakingSystem | DefaultNullOrEmpty }}
-      </div>
+    <dd id="test-roadFriendly" class="govuk-summary-list__value">
+      {{ activeRecord.roadFriendly | DefaultNullOrEmpty }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Brake actuator
+      Vehicle class
     </dt>
-    <dd id="test-brakeActuator" class="govuk-summary-list__value">
-      {{ activeRecord.axles.slice(-1)[0].brakes.brakeActuator | DefaultNullOrEmpty }}
+    <dd id="test-vehicleClassDescription" class="govuk-summary-list__value">
+      {{
+        techRecHelpers.isNullOrEmpty(activeRecord.vehicleClass)
+          ? '-'
+          : (activeRecord.vehicleClass.description | DefaultNullOrEmpty).charAt(0).toUpperCase() +
+            (activeRecord.vehicleClass.description | DefaultNullOrEmpty).substr(1).toLowerCase()
+      }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Lever length
+      Vehicle configuration
     </dt>
-    <dd id="test-leverLength" class="govuk-summary-list__value">
-      <div *ngIf="activeRecord.axles[1] != null; else hyphen">
-        <span>{{ activeRecord.axles[1].brakes.leverLength | DefaultNullOrEmpty }}</span>
-      </div>
+    <dd id="test-vehicleConfiguration" class="govuk-summary-list__value">
+      {{
+        (activeRecord.vehicleConfiguration | DefaultNullOrEmpty).charAt(0).toUpperCase() +
```

Comment:
used again here, maybe it is a pipe or util function we reuse.
@alleypa @irina005 @rspasici thoughts?

Reply:
removed the tecRecHelper and moved to a function.